### PR TITLE
Indexing articles

### PR DESCRIPTION
**Pull request by Lauraa1003**
https://indexing--www--headwire-edge-delivery.hlx.page/

### CODE REVIEW

File: helix-query.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+version: 1
+
+indices:
+  articles:
+    include:
+      - '/articles/**'
```
Comment (icaraps) [inline helix-query.yaml]: @Lauraa1003  Do we want to keep articles ? or name it blog ?

File: helix-query.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    target: /query-index.xlsx
+    properties:
```
Comment (icaraps) [inline helix-query.yaml:8]: We should probably add image, description and maybe keywords to be on the safe side

File: helix-query.yaml (hunk 1 above)
Comment (Lauraa1003) [reply helix-query.yaml]: Named it blog

File: helix-query.yaml (hunk 2 above)
Comment (Lauraa1003) [reply helix-query.yaml:8]: added extra properties

**Pull request merged**